autoflip_calculators/text_detection_calculator.cc

```diff
@@ -59,28 +72,32 @@ class TextDetectionCalculator : public CalculatorBase {
   ::mediapipe::Status Process(mediapipe::CalculatorContext* cc) override;
 
  private:
-  double NormalizeX(const int pixel);
-  double NormalizeY(const int pixel);
   // Decode the outputs of the EAST neural network
   void DecodeBoundingBoxes(const cv::Mat& socres, const cv::Mat& geometry, float scoreTresh,
-                         std::vector<RotatedRect>& detections, std::vector<float>& confidences);
+                         std::vector<cv::RotatedRect>& detections, std::vector<float>& confidences);
+
+  // Detect the text.
+  void DetectText(const cv::Mat& frame, cv::dnn::Net detector,
+                cv::Mat& scores, cv::Mat& geometry);
 
   // Calculator options.
   TextDetectionCalculatorOptions options_;
 
   // A scorer used to assign weights to texts.
   std::unique_ptr<VisualScorer> scorer_;
 
-  // Dimensions of video frame
-  int frame_width_;
-  int frame_height_;
+  // Threshold to determine whether a target is text,
+  // if greater than kConfidence_threshold_, it's text.
+  const float kConfidence_threshold_ = 0.5;
+  
+  // Threshold for non maximum supression.
+  const float kNMS_threshold_ = 0.4;
 
-  const float confidence_threshold = 0.7; //0.5;
-  const float nms_threshold = 0.4;
-  const std::string model_path = "/mediapipe/models/frozen_east_text_detection.pb";
-  cv::dnn::Net detector;
-  std::vector<Mat> outs;
-  std::vector<String> outNames(2);
+  // Text detection model path.
+  cv::String kModel_path_ = "/usr/local/google/home/zzhencchen/mediapipe/mediapipe/models/frozen_east_text_detection.pb";
```

Comment:
This will need to be a relative path so it will work for other users.  Also, consider putting the model path in the options proto so it can be changed without recompiling.